### PR TITLE
feat(EvmEventForm): hide asset price for informational events

### PR DESCRIPTION
**Pull request by cleanerzkp**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/src/components/history/events/forms/EvmEventForm.vue [hunk 1]
```diff
@@ -55,6 +55,8 @@ const errorMessages = ref<Record<string, string[]>>({});
 
 const externalServerValidation = () => true;
 
+const isInformationalEvent = computed(() => eventType.value === 'informational');
```
Comment (lukicenturi) [inline frontend/app/src/components/history/events/forms/EvmEventForm.vue:58]: use `get()`

File: frontend/app/src/components/history/events/forms/EvmEventForm.vue [hunk 2]
```diff
@@ -228,10 +230,15 @@ async function save(): Promise<boolean> {
     eventType: get(eventType),
     eventSubtype: get(eventSubtype),
     asset: get(asset),
-    balance: {
-      amount: get(numericAmount).isNaN() ? Zero : get(numericAmount),
-      usdValue: get(numericUsdValue).isNaN() ? Zero : get(numericUsdValue),
-    },
+    balance: isInformationalEvent.value
```
Comment (lukicenturi) [inline frontend/app/src/components/history/events/forms/EvmEventForm.vue:233]: here too

File: frontend/app/src/components/history/events/forms/HistoryEventAssetPriceForm.vue [hunk 3]
```diff
@@ -18,9 +18,11 @@ const props = withDefaults(
     usdValue: string;
     disableAsset?: boolean;
     v$: Validation;
+    hidePriceFields: boolean;
```
Comment (lukicenturi) [inline frontend/app/src/components/history/events/forms/HistoryEventAssetPriceForm.vue:21]: hidePriceFields?: boolean

Comment (cleanerzkp) [reply frontend/app/src/components/history/events/forms/HistoryEventAssetPriceForm.vue:21]: if we want to show them or not

File: frontend/app/src/components/history/events/forms/EvmEventForm.vue [hunk 4]
```diff
@@ -228,10 +230,15 @@ async function save(): Promise<boolean> {
     eventType: get(eventType),
     eventSubtype: get(eventSubtype),
     asset: get(asset),
-    balance: {
-      amount: get(numericAmount).isNaN() ? Zero : get(numericAmount),
-      usdValue: get(numericUsdValue).isNaN() ? Zero : get(numericUsdValue),
-    },
+    balance: get(isInformationalEvent)
+      ? {
+          amount: Zero,
```
Comment (lukicenturi) [inline frontend/app/src/components/history/events/forms/EvmEventForm.vue:235]: amount should not be zero, only the usdValue

**Comment by lukicenturi** [comment]
For type "History Event", please also move this to the top, so it's consistent 
image

**Comment by lukicenturi** [comment]
Also @cleanerzkp in the `HistoryEventAssetPriceForm.vue`, you need to make sure the price fetching didn't happen.
Hiding the UI is one thing, but this price fetching is still happens. This should be the main point of this PR

File: frontend/app/src/components/history/events/forms/HistoryEventAssetPriceForm.vue [hunk 5]
```diff
@@ -160,6 +165,8 @@ async function fetchHistoricPrices() {
 }
 
 watch([datetime, asset], async () => {
+  if (get(hidePriceFields))
+    return;
```
Comment (lukicenturi) [inline frontend/app/src/components/history/events/forms/HistoryEventAssetPriceForm.vue:169]: This is redundant check, isn't it? You also check it inside `fetchHistoricPrices()`

**Comment by lukicenturi** [comment]
while the one with type 'evm event' the space under asset is a little bit too much
image

File: frontend/app/src/components/history/events/forms/HistoryEventAssetPriceForm.vue (hunk 5 above)
Comment (lukicenturi) [reply frontend/app/src/components/history/events/forms/HistoryEventAssetPriceForm.vue:169]: also make it part of the watch.
You need to test, by set it first to informational,
and then set it to other.
and vice versa.
make sure the price is always fetching when it's not informational, even if we change the event type.

**Review by lukicenturi — APPROVED** [review]
LGTM!

**Pull request merged**